src/charm.py

```diff
@@ -227,15 +235,15 @@ def _service_name(self):
         return self._get_config_or_relation_data("service-name", "")
 
     @property
-    def _service_port(self):
+    def _service_port(self) -> Any:
```

Comment:
Isn't it `int`? `Any` is kind of pointless since it essentially disables typing checks, perhaps we make a follow up story to clean up the type hints?

Reply:
My bad, is int, thanks.
A follow up story would be great also considering the plugin to check for Kubernetes types.

Reply:
ISD-422 was created to address type hints.